lib/pf/UnifiedApi.pm

```diff
@@ -1914,6 +1914,7 @@ sub setup_api_v1_dynamic_reports_routes {
     my $resource_route = $root->under("/dynamic_report/#report_id")->to(controller => $controller, action => "resource")->name("${name}.resource");
```

Comment:
As @satkunas pointed out, these should now be called `/api/v1/reports`

We need to publish an upgrade note for that too and since you reworked most of the API endpoints, it probably worth pointing out that the input/output format has also changed